okta/config.go

```diff
@@ -182,6 +190,99 @@ func oktaSDKClient(c *Config) (client *sdk.Client, err error) {
 	return
 }
 
+// TODU remove err???
+func oktaV3SDKClient(c *Config) (client *okta.APIClient, err error) {
+	var httpClient *http.Client
+	logLevel := strings.ToLower(os.Getenv("TF_LOG"))
+	debugHttpRequests := (logLevel == "1" || logLevel == "debug" || logLevel == "trace")
+	if c.backoff {
+		retryableClient := retryablehttp.NewClient()
+		retryableClient.RetryWaitMin = time.Second * time.Duration(c.minWait)
+		retryableClient.RetryWaitMax = time.Second * time.Duration(c.maxWait)
+		retryableClient.RetryMax = c.retryCount
+		retryableClient.Logger = c.logger
+		if debugHttpRequests {
+			// Needed for pretty printing http protocol in a local developer environment, ignore deprecation warnings.
+			//lint:ignore SA1019 used in developer mode only
+			retryableClient.HTTPClient.Transport = logging.NewTransport("Okta", retryableClient.HTTPClient.Transport)
+		} else {
+			retryableClient.HTTPClient.Transport = logging.NewSubsystemLoggingHTTPTransport("Okta", retryableClient.HTTPClient.Transport)
+		}
+		retryableClient.ErrorHandler = errHandler
+		retryableClient.CheckRetry = checkRetry
+		httpClient = retryableClient.StandardClient()
+		c.logger.Info(fmt.Sprintf("running with backoff http client, wait min %d, wait max %d, retry max %d", retryableClient.RetryWaitMin, retryableClient.RetryWaitMax, retryableClient.RetryMax))
+	} else {
+		httpClient = cleanhttp.DefaultClient()
+		if debugHttpRequests {
+			// Needed for pretty printing http protocol in a local developer environment, ignore deprecation warnings.
+			//lint:ignore SA1019 used in developer mode only
+			httpClient.Transport = logging.NewTransport("Okta", httpClient.Transport)
+		} else {
+			httpClient.Transport = logging.NewSubsystemLoggingHTTPTransport("Okta", httpClient.Transport)
+		}
+		c.logger.Info("running with default http client")
+	}
+
+	// adds transport governor to retryable or default client
+	if c.maxAPICapacity > 0 && c.maxAPICapacity < 100 {
+		c.logger.Info(fmt.Sprintf("running with experimental max_api_capacity configuration at %d%%", c.maxAPICapacity))
+		apiMutex, err := apimutex.NewAPIMutex(c.maxAPICapacity)
+		if err != nil {
+			return nil, err
+		}
+		httpClient.Transport = transport.NewGovernedTransport(httpClient.Transport, apiMutex, c.logger)
+	}
+	var orgUrl string
+	var disableHTTPS bool
+	if c.httpProxy != "" {
+		orgUrl = strings.TrimSuffix(c.httpProxy, "/")
+		disableHTTPS = strings.HasPrefix(orgUrl, "http://")
+	} else {
+		orgUrl = fmt.Sprintf("https://%v.%v", c.orgName, c.domain)
+	}
+
+	setters := []okta.ConfigSetter{
+		okta.WithOrgUrl(orgUrl),
+		okta.WithCache(false),
+		// TODU
+		// okta.WithHttpClientPtr(httpClient),
```

Comment:
Why aren't we setting the httpClient in the Okta SDK v3 client set up?

Reply:
Because there is not a Config Setter for that. I am updating that in v3.0.2 of Golang sdk, along with the remove the Get in Getter